storage/innobase/dict/dict0stats.cc

```diff
@@ -1840,23 +1933,28 @@ dict_stats_scan_page(
 	return(offsets_rec);
 }
 
-/** Dive below the current position of a cursor and calculate the number of
-distinct records on the leaf page, when looking at the fist n_prefix
-columns. Also calculate the number of external pages pointed by records
-on the leaf page.
-@param[in]	cur			cursor
-@param[in]	n_prefix		look at the first n_prefix columns
-when comparing records
-@param[out]	n_diff			number of distinct records
-@param[out]	n_external_pages	number of external pages
-@return number of distinct records on the leaf page */
+/** Dive below the current position of a cursor and calculate
+the number of distinct records on the leaf page, when looking at
+the first n_prefix columns. Also calculate the number of external
+pages pointed by records on the leaf page.
+@param[in]	cur		cursor
+@param[in]	n_prefix	look at the first n_prefix columns
+				when comparing records
```

Comment:
I don’t think we should use any `[in]` or `[out]` in new code. `const` qualifiers or passing-by-value already indicate this.
```
@param n_prefix  number of fields to compare
```